### ISSUE TITLE
Request for .deb Linux package 

### ISSUE THREAD

**Issue by kahn-msupply** [msupply-foundation/open-msupply#5129]
## Is your feature request related to a problem? Please describe 👀

This request is regarding installations for Linux based systems, most commonly used would be Ubuntu Server for running Open mSupply Central Server.

This feature would be great for the future, as clients start to migrate to Open mSupply.

## Describe the solution you'd like 🎁

.deb package for installing Open mSupply Central Server on Ubuntu Servers, without the need to compile from source.

### Additional context 💌

It may be worth outlining an expected installation process e.g. using venv

### Moneyworks Jobcode 🧰

~

**Comment by andreievg** [comment]
Not putting milestone in for now, waiting for time and urgency to be bumped, we can build manually for now if required

**Comment by kahn-msupply** [comment]
For the .deb builds, I imagine we will possibly need to compile for different CPU architectures and distros in future. For the BES deployment, it looks like they are using Raspberry Pi's with Debian, so we will need ARM64 build. I think we should consider creating a build for AMD64 as well, if not too much work?

**Comment by Chris-Petty** [comment]
Yes using docker, the immediate need for this in March is no more!
